etcdserver/api/v3rpc/util.go

```diff
@@ -81,3 +81,12 @@ func togRPCError(err error) error {
 	}
 	return grpcErr
 }
+
+func isClientCtxErr(ctxErr error, err error) bool {
+	if ctxErr != nil {
```

Comment:
```go
if ctxErr == nil {
    return nil
}
if ctxErr == context.Canceled || ctxErr== context.DeadlineExceeded {
    return true
}
ev, err := status.FromError(err)
if err != nil {
    return false
}
code := ev.Code()
return code == codes.Canceled || code == codes.DeadlineExceeded
```

Reply:
On ```ctxErr != nil, return true``` (instead of ctxErr == context.Canceled || ctxErr == context.DeadlineExceeded) makes sense since context errors seems either ```context.Canceled``` or ```context.DeadlineExceeded```